docs/victoriametrics-cloud/get-started/quickstart.md

```diff
@@ -59,7 +59,7 @@ When creating a deployment, the following options are available:
 | **`Deduplication`** | Deduplication handles redundant data in high-availability (HA) setups to retain only one sample per interval. For best results, set deduplication to match the collect metrics interval. If you have multiple intervals, set it to the shortest one. |
 | <nobr>**`Maintenance Window`**</nobr> | We use this value as the preferred window for us to perform maintenance operations, such as upgrades, when needed. |
 
-![Selecting a tier](create_deployment_form_down.webp "Selecting a tier")
+![Selecting a tier](/victoriametrics-cloud/get-started/create_deployment_form_down.webp "Selecting a tier")
```

Comment:
use absolute urls it's much easy to maintain

Reply:
```suggestion
![Selecting a tier](https://docs.victoriametrics.com/victoriametrics-cloud/get-started/create_deployment_form_down.webp "Selecting a tier")
```